src/_posts/2020-02-20-remote-pairing-a-true-story.md

```diff
@@ -0,0 +1,68 @@
+
+The challenge goes beyond those inherent to the project: is key to establish the new team dynamics, how the work is split, how requirements are refined and the workload is handled.
+
+There are several pillars to build a team upon, if you want to leverage the chances your project to succeed, undoubtedly embracing [XP Practies](http://www.extremeprogramming.org/) should be in your table, and engaging any agile flavour to handle the business delivery workload is for sure in the handbook of a good project coordinator.
+
+## The Challenge
+Now imagine the team is composed of skilled and experienced members, but they are in different locations, or perhaps working remote, or working alternate days from home, or even in different time zone.
+
+This is a perfectly valid team set up in our brand new highly connected world, and defy the traditional ways of working, and put our engineering practices under the loupe, make us thrive to find the best to coupe with the new context, but still using the best known practices in its essence.
+ 
+## Pair Programming
+Pair programming is considered a Core Practice among those 12 practices enumerated within the XP scope, and is basically a social activity, is the point where two developers join to work together in the same tool they use to produce their craft, create enriched code that emerges from their interaction.
+
+Working in pairs relates deeply with other core practices such as [Code Ownership](https://ronjeffries.com/xprog/what-is-extreme-programming/#collective), [Code Standards](https://ronjeffries.com/xprog/what-is-extreme-programming/#coding) and even helps build the [Metaphor](https://ronjeffries.com/xprog/what-is-extreme-programming/#metaphor) of the system across the team.
+
+When you and the brand new team agree you'll embrace pair programming as a whole, all code to be sent into production is created by two people working together at a single computer.
+
+The best way to pair program is to just sit side by side in front of the monitor. [Slide the keyboard and mouse back and forth.](http://www.extremeprogramming.org/rules/pair.html)
+
+But, how does this last statement matches with the context presented in the paragraph above?
+
+## Pairing Remote is still pairing
+Pair Programming is a demanding activity and remote pairing is even more demanding, but it doesn't need to be even more exhausting, while it also can be done effectively. Here there is a compound of lessons learned that might help your team boost their pair programming experience:
+
+- Requires good and interrupted connections
+- Get comfortable headset for your team, avoid microphones that capture too much environmental noise.
+- There are dozens of tools available, but stick to a video conference that allows only one driver.
+- Ask questions and be prepared to explain, keep your pair engaged
+- Is not that simple to say "here" or "there", use the proper method or functions names, use the line numbers to reference a location in the file.
```

Comment:
This could read - 

> It is as simple to say "here" or "there" when referring to something to change or add. Use the method or functions names and line numbers to reference a location in the file.